src/Avalonia.Themes.Default/CalendarDatePicker.xaml

```diff
@@ -96,7 +98,7 @@
                    DataValidationErrors.Errors="{TemplateBinding (DataValidationErrors.Errors)}"
                    VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
                    HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
-                   Grid.Column="0"/>
+                   Grid.Column="0" />
 
           <Button Name="PART_Button"
```

Comment:
Please revert all the whitespace changes in this file, it will mess up the history and we wont be able to trace who made what changes later. I cant see which is the actual change here if any...

<DataValidationErrors> element must be present here somewhere.

Reply:
> DataValidationErrors element must be present here somewhere. 

I use the one ot the TextBox as this looks really neat imo (sorry for the german message): 
![image](https://user-images.githubusercontent.com/47110241/148176149-7a0cb38d-9c1d-473c-9f67-bf6e29d55c9e.png)

The whitespaces are removed now.